### PR TITLE
Remove sql file support

### PR DESCRIPTION
**Pull request by Ekrekr**

### CODE REVIEW

**Comment by lewish** [comment]
You can remove SQL from here too: https://github.com/dataform-co/dataform/blob/main/sandbox/vm/compile.ts#L47

**Review by lewish — APPROVED** [review]

File: version.bzl [hunk 1]
```diff
@@ -1,3 +1,3 @@
 # NOTE: If you change the format of this line, you must change the bash command
 # in /scripts/publish to extract the version string correctly.
-DF_VERSION = "2.1.0"
```
Comment (BenBirt) [inline version.bzl:3]: hm. should we now be making a `main_v2` to be able to push bugfixes (security fixes) to v2 users?

Comment (Ekrekr) [reply version.bzl:3]: Fair point, more effort than it's worth currently - will hold off submitting this

**Comment by Ekrekr** [comment]
Actually done in https://github.com/dataform-co/dataform/pull/1564

**Pull request closed without merging**